benchto-service/src/main/java/io/trino/benchto/service/model/Metric.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.benchto.service.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.hibernate.annotations.BatchSize;
+import org.hibernate.annotations.Cache;
+
+import javax.persistence.CollectionTable;
+import javax.persistence.Column;
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.MapKeyColumn;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+import java.io.Serializable;
+import java.util.Map;
+import java.util.Objects;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static com.google.common.collect.Maps.newHashMap;
+import static javax.persistence.FetchType.EAGER;
+import static org.hibernate.annotations.CacheConcurrencyStrategy.TRANSACTIONAL;
+
+@Entity
+@Table(name = "metrics")
+public class Metric
+        implements Serializable
+{
+    @Id
+    @SequenceGenerator(name = "metrics_id_seq",
+            sequenceName = "metrics_id_seq",
+            allocationSize = 1)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE,
+            generator = "metrics_id_seq")
+    @Column(name = "id")
+    @JsonIgnore
+    private long id;
+
+    @Size(min = 1, max = 64)
+    @Column(name = "name")
+    private String name;
+
+    @NotNull
+    @Enumerated(EnumType.STRING)
+    @Column(name = "unit")
+    private MeasurementUnit unit;
+
+    @Cache(usage = TRANSACTIONAL)
+    @BatchSize(size = 10)
+    @ElementCollection(fetch = EAGER)
+    @MapKeyColumn(name = "name")
+    @Column(name = "value")
+    @CollectionTable(name = "metric_attributes", joinColumns = @JoinColumn(name = "metric_id"))
+    private Map<String, String> attributes = newHashMap();
```

Comment:
This will probably make it harder to extract specific attributes for a measurement (min, max). This might be a problem when someone is writing a script.

Could we embed attributes somehow for a measurment?
It seems that measurement for a given type will have a fixed list of attributes, so maybe we don't need flexibility of extra `metric_attributes` table.